cli/src/commands/prev.rs

```diff
@@ -78,25 +76,15 @@ pub(crate) fn cmd_prev(
             .view()
             .heads()
             .contains(current_wc_id);
-    let current_wc = workspace_command.repo().store().get_commit(current_wc_id)?;
-    let start_id = if edit {
-        current_wc_id
-    } else {
-        match current_wc.parent_ids() {
-            [parent_id] => parent_id,
-            _ => return Err(user_error("Cannot run `jj prev` on a merge commit")),
-        }
-    };
-    let ancestor_expression = RevsetExpression::commit(start_id.clone()).ancestors_at(offset);
+    // If --edit is specified or implied, then the offset is relative to the
+    // current working copy commit. Otherwise, the offset is relative to the
+    // parent(s) of the current wc commit.
```

Comment:
nit: I think the comment in `cmd_next()` is clearer.
```
    // If we're editing, start at the working-copy commit. Otherwise, start from
    // its direct parent(s).
```

Reply:
Updated.